spec/models/creation/repository_spec.rb

```diff
@@ -41,9 +41,7 @@
     let(:cup_cake) { create(:creation, name: 'Cup Cake') }
 
     it "returns cakes with a matching name" do
-      results = subject.search('cake')
-      expect(results).to include(cake)
-      expect(results).to_not include(cup_cake)
+      expect(subject.search('cake')).to match_array([cake])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.